packages/apps/src/Apps.tsx

```diff
@@ -199,7 +197,7 @@ export default React.memo(styled(Apps)`
 
   &.menu-open {
     .apps--SideBar-Wrapper {
-      width: 12rem;
+      max-width: 230px;
```

Comment:
Would really prefer to stick to rem since it is the way it is applied everywhere. (Same for padding, etc.) Only fallback to px when there is no other choice.